Scripts

Travior opened the pull request.

Travior committed:
Add files via upload

Script to enable ipv4 and ipv6 forwarding

Travior committed:
Update ipforward.py

Travior committed:
Update ipforward.py

Jartreg commented on ipforward.py:
```diff
@@ -0,0 +1,50 @@
+    print("   _| |_| |         | || (_) | |   \ V  V / (_| | | | (_| | | | | | (_| |")
+    print("   \___/\_|         \_| \___/|_|    \_/\_/ \__,_|_|  \__,_|_|_| |_|\__, |")
+    print("                                                                    __/ |")
+    print("                                                                   |___/ ")
```
Ist das wirklich nötig?

Travior replied:
Ja wirklich alle scripts nehmen so etwas als überschrift. Ist eine alte kali manier ;)

Jartreg reviewed (commented):
Das ist ja eigentlich erstmal nur für die Präsentation gedacht... Wär's nicht besser, du machst das so, dass du das für die Beispiele nur auszuführen brauchst?

Jartreg reviewed (changes requested):
Am Besten wär's vielleicht noch, wenn du das in einen Unterordner tun würdest.

Travior committed:
Update ipforward.py

Now featuring multiple pages
Extended Menu, to undo changes

Travior reviewed (commented):
Now featuring multiple pages
Extended Menu, to undo changes

Travior commented:
> Das ist ja eigentlich erstmal nur für die Präsentation gedacht... Wär's nicht besser, du machst das so, dass du das für die Beispiele nur auszuführen brauchst?

Das ist es ja erstmal, macht ja keinen großen unterschied ob ich jetzt noch 2 Tasten drücke oder nicht. Ist halt nur vielleicht für das Verständnis einfacher

Jartreg commented on ipforward.py:
```diff
@@ -0,0 +1,107 @@
+switchPage = False
+page = 1
+
+endlessrun = True
```
Die Variable `endlessrun` wird doch eigentlich nicht gebraucht, oder? Warum nicht
```python
while True:
```

Jartreg commented on ipforward.py:
```diff
@@ -0,0 +1,107 @@
+option4 = False
+option5 = False
+option6 = False
+switchPage = False
```
Die Variable `switchPage` ist unnötig

Jartreg commented on ipforward.py:
```diff
@@ -0,0 +1,107 @@
+option3 = False
+option4 = False
+option5 = False
+option6 = False
```
Diese Namen sind doch nicht dein Ernst, oder?

Jartreg commented on ipforward.py:
```diff
@@ -0,0 +1,107 @@
+        option = str(optioninput)
+        if option == "1":
+            output = subprocess.check_output(['bash', '-c', ipv4command])
+            option1 = True
```
Kannst du [Zeile 38](https://github.com/Jartreg/prometeus-https-demo/pull/2/files#diff-3d8b37ff3bf410808455cc34849671c9R38) nicht schon hierhin tun? Dann sind diese ganzen optionN-Variablen nicht mehr nötig...

Jartreg commented on ipforward.py:
```diff
@@ -0,0 +1,107 @@
+            page = 1
+        elif option == "2":
+            output = subprocess.check_output(['bash', '-c', ipv6Command])
+            option2 = True
```
Das gleiche hier mit [Zeile 41](https://github.com/Jartreg/prometeus-https-demo/pull/2/files#diff-3d8b37ff3bf410808455cc34849671c9R41)

Jartreg commented on ipforward.py:
```diff
@@ -0,0 +1,107 @@
+        elif option == 'n' or option == 'N':
+            page = 2
+        else:
+            option3 = True
```
Das gleiche hier mit [Zeile 44](https://github.com/Jartreg/prometeus-https-demo/pull/2/files#diff-3d8b37ff3bf410808455cc34849671c9R44)

Jartreg commented on ipforward.py:
```diff
@@ -0,0 +1,107 @@
+        option = str(optioninput)
+        if option == "1":
+            output = subprocess.check_output(['bash', '-c', ipv4commandTwo])
+            option4 = True
```
Siehe [Zeile 81](https://github.com/Jartreg/prometeus-https-demo/pull/2/files#diff-3d8b37ff3bf410808455cc34849671c9R81)

Jartreg commented on ipforward.py:
```diff
@@ -0,0 +1,107 @@
+            page = 2
+        elif option == "2":
+            output = subprocess.check_output(['bash', '-c', ipv6CommandTwo])
+            option5 = True
```
Siehe [Zeile 84](https://github.com/Jartreg/prometeus-https-demo/pull/2/files#diff-3d8b37ff3bf410808455cc34849671c9R84)

Jartreg commented on ipforward.py:
```diff
@@ -0,0 +1,107 @@
+        elif option == 'p' or option == 'P':
+            page = 1
+        else:
+            option6 = True
```
Siehe [Zeile 87](https://github.com/Jartreg/prometeus-https-demo/pull/2/files#diff-3d8b37ff3bf410808455cc34849671c9R87)

Jartreg commented on ipforward.py:
```diff
@@ -0,0 +1,107 @@
+        print("   _| |_| |         | || (_) | |   \ V  V / (_| | | | (_| | | | | | (_| |")
+        print("   \___/\_|         \_| \___/|_|    \_/\_/ \__,_|_|  \__,_|_|_| |_|\__, |")
+        print("                                                                    __/ |")
+        print("                                                                   |___/ ")
```
Du wiederholst hier das gesamte Banner... Tu das doch in 'ne Funktion

Jartreg reviewed (changes requested):
Hier noch ein paar Verbesserungsvorschläge was die Schönheit angeht. Zudem sollte das das gesamte Projekt bis auf den Code auf deutsch sein. Daher wäre es gut, wenn du die englischen Ausgaben ("Wrong parameter" usw...) durch deutsche ersetzen könntest...

Travior committed:
bewege ipforward.py in extra Ordner

Travior committed:
Vereinfache den Code
* entferne endlessrun = True
* bewege das Banner in eine eigene Funktion

Travior committed:
Erstelle ARP-Spoof Skript (Ziel --> Router)
 *Erstelle automatisch ip.txt im Verzeichnis
 *Schreibe IP-Adressen Automatisch in ip.txt

Jartreg commented on scripts/arpspoof/arpspoof_main.py:
```diff
@@ -0,0 +1,73 @@
+    if wrongParameterVar==True:
+        print("Wrong Parameter")
+    else:
+        pass
```
Unnötiger `else`-Zweig

Travior committed:
Entferne unnötigen Else Zweig
